### ISSUE TITLE
graphql-anywhere produces ts typing error 

### ISSUE THREAD

**Issue by iamdanthedev** [apollographql/apollo-client#2806]
Hi

I was getting the following error, despite the fact that node_modules is excluded in tsconfig. 

The solution was to delete `src` folder from `node_modules/graphql-anywhere`. I suppose it should not be there anyway

```
ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts
[tsl] ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts(2,3)
      TS2305: Module '"/home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql"' has no exported member 'DocumentNode'.

ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts
[tsl] ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts(3,3)
      TS2305: Module '"/home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql"' has no exported member 'SelectionSetNode'.

ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts
[tsl] ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts(4,3)
      TS2305: Module '"/home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql"' has no exported member 'FieldNode'.

ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts
[tsl] ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts(5,3)
      TS2305: Module '"/home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql"' has no exported member 'FragmentDefinitionNode'.

ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts
[tsl] ERROR in /home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql.ts(6,3)
      TS2305: Module '"/home/danthedev/Projects/ypc/ypc-app/node_modules/graphql-anywhere/src/graphql"' has no exported member 'InlineFragmentNode'.
```

**Version**
    "apollo-client-preset": "^1.0.5",
    "apollo-codegen": "^0.18.3",
    "apollo-link-batch-http": "^1.0.1",
    "apollo-link-context": "^1.0.3",
    "apollo-link-error": "^1.0.3",
    "apollo-server-express": "^1.1.7",

**Comment by rasdaniil** [comment]
The solution is quite simple please look here: https://github.com/apollographql/apollo-client/pull/2807

**Comment by shafqatdotnet** [comment]
i am facing the exact same issue. Is there any update on releasing fix in #2806?

**Comment by katzo9** [comment]
This bug is back: the src directory is not longer present in https://github.com/apollographql/apollo-client/blob/master/packages/graphql-anywhere/.npmignore
